demos/LowAllocationWebServer/ByteSpanExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace LowAllocationServer
+{
+    public static class ByteSpanExtensions
```

Comment:
@krwq, could you see which of these should be in the Slices library? Or maybe you already have these.